CHANGELOG.md

```diff
@@ -12,11 +12,12 @@ The format is loosely based on [Keep a Changelog](http://keepachangelog.com/en/1
 This release contains several **minor breaking changes**. Please review your code with respect to the following:
 
 * All custom argument matcher methods (including those using `Match.Create<T>`) must now be marked with the `[Matcher]` attribute. For this reason, `MatcherAttribute` is no longer marked obsolete (@stakx, #732)
-* Method overload resolution for `mock.Protected().Setup("VoidMethod", ...)` may change due to a new overload: If you set up a `void` method whose first argument is a `bool`, make sure that argument gets interpreted as part of `args`, not as `exactParameterMatch` (see also *Added* section below). (@stakx, #751)
+* Method overload resolution for `mock.Protected().Setup("VoidMethod", ...)` and `mock.Protected().Verify(...)` may change due to a new overload: If you set up a `void` method whose first argument is a `bool`, make sure that argument gets interpreted as part of `args`, not as `exactParameterMatch` (see also *Added* section below). (@stakx & @Shereef, #751, #753)
```

Comment:
Please note that the description is no longer quite correct. I'd suggest you fix this by rephrasing the following:

* We're talking about more than just one new method overload. Therefore:

   ```diff
   -due to a new overload:
   +due to new overloads:
   ```

* We're no longer taking just about setting up `void` methods; it's now also about verifying any type of method. Therefore let's be less specific:

   ```diff
   -If you set up a `void` method whose first argument is a `bool`,
   +If the first argument is a `bool`, 
   ```

Reply:
Done, thanks for your attention to detail